model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/records/RecordType.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 Basis Technology Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.basistech.rosette.apimodel.recordsimilarity.records;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Arrays;
+
+@AllArgsConstructor
+@Getter
+public enum RecordType {
```

Comment:
I would think this should be FieldType (or RecordFieldType if we're worried that sounds too ambiguous), since it's the type of a field, where a record has multiple fields